hatchet_sdk/worker/worker.py

```diff
@@ -155,6 +162,39 @@ def setup_loop(self, loop: asyncio.AbstractEventLoop | None = None) -> bool:
             created_loop = True
             return created_loop
 
+    async def health_check_handler(self, request: Request) -> Response:
+        status = self.status()
+
+        return web.json_response({"status": status.name})
+
+    async def metrics_handler(self, request: Request) -> Response:
+        self.worker_status_gauge.set(1 if self.status() == WorkerStatus.HEALTHY else 0)
```

Comment:
consideration:

should we consider 1 for healthy and -1 for unhealthy. thought process here is we may want to change this guage to monitor num available slots where 0 can still represent a healthy worker that is just busy

alternatively this might be a separate guage in the future

Reply:
I think I'd prefer the -1 since we won't need to change in the future

Reply:
Just catching up here, but I think -1 might be problematic -- the query to determine if there are any healthy workers is going to be a `sum` in Prometheus which is going to cause unexpected behavior if there are -1s.

Reply:
Nice catch! I didn't know this - I'll revert it back to 0 for now 👍